Split/Network/Streaming/SseNotifications.swift

```diff
@@ -108,29 +108,40 @@ struct ControlNotification: NotificationTypeField {
 
 /// Indicates change in MySegments
 struct MySegmentsUpdateNotification: NotificationTypeField {
-    private (set) var type: NotificationType
+    var type: NotificationType {
+        return .mySegmentsUpdate
+    }
     let changeNumber: Int64
     let includesPayload: Bool
     let segmentList: [String]?
 }
 
 /// Indicates that a Split was killed
 struct SplitKillNotification: NotificationTypeField {
-    private (set) var type: NotificationType
+    var type: NotificationType {
+        return .splitKill
+    }
     let changeNumber: Int64
     let splitName: String
     let defaultTreatment: String
 }
 
 /// indicates Split changes
 struct SplitsUpdateNotification: NotificationTypeField {
-    private (set) var type: NotificationType
+    var type: NotificationType {
+        return .splitUpdate
+    }
     let changeNumber: Int64
 }
 
 /// Indicates a notification related to occupancy
 struct OccupancyNotification: NotificationTypeField {
-    private (set) var type: NotificationType = .occupancy
+    private let kControlPriToken = "control_pri"
+    private let kControlSecToken = "control_sec"
+    var channel: String?
```

Comment:
shouldn't `channel` be mandatory?

Reply:
The channel field doesn´t exists in the json message. So, to avoid crashes when parsing the field is optional.